dashboard/app/views/ops_mailer/_exit_survey_url.html.haml

```diff
@@ -1,52 +1,63 @@
-- case workshop.program_type.to_i
-  - when 1
-    %p
-      We appreciate any feedback you have. Please take a moment to complete this
-      - case workshop.phase
-        - when 2
+%p
+  We appreciate any feedback you have. Please take a moment to complete this
+  - case workshop.program_type_short_name
+    - when 'CS in Science'
+      - case workshop.phase_short_name
+        - when 'Phase 2'
           %a{href:"https://docs.google.com/a/code.org/forms/d/1QG9eCbKJD26UNvTC0C9ZZyrp63WjzQSK5gQPP4lsZ2c/viewform"}
             #{link_text}
-          about your Code.org workshop experience.
-        - when 4
+        - when 'Phase 3B'
+          %a{href:"https://docs.google.com/a/code.org/forms/d/1V4sm9cPvABp4ipo4MIMUq9cf-aa-sFWafI5qjrlSV8M/viewform?c=0&w=1"}
+            #{link_text}
+        - when 'Phase 4'
           %a{href:"https://docs.google.com/a/code.org/forms/d/106j69amXoeZkBEXfNszLUpIx3FfyVcOEdo5Ce6oKMA4/viewform?c=0&w=1&usp=send_form"}
             #{link_text}
-          about your Code.org workshop experience.
-  - when 2
-    %p
-      We appreciate any feedback you have. Please take a moment to complete this
-      - case workshop.phase
-        - when 2
+    - when 'CS in Algebra'
+      - case workshop.phase_short_name
+        - when 'Phase 2'
           %a{href:"https://docs.google.com/a/code.org/forms/d/1kEkyOk-Y4MsWVVjmkaw-UDfCE4Wl2Ol3MTrCw4IW8vw/viewform"}
             #{link_text}
-        - when 4
+        - when 'Phase 3B'
+          %a{href:"https://docs.google.com/a/code.org/forms/d/12koHnpyF2azrNNsEgvy66_dqXXmWeRmEJ9ICJA-gT-M/viewform?c=0&w=1"}
+            #{link_text}
+        - when 'Phase 4'
           %a{href:"https://docs.google.com/a/code.org/forms/d/1oBYFuv1xlInKD7asJ7gR4GA1xbsdKDjG1PI4tvgdwWc/viewform?c=0&w=1"}
             #{link_text}
-          about your Code.org workshop experience.
-  - when 3
-    %p
-      We appreciate any feedback you have. Please take a moment to complete this
-      - case workshop.phase
-        - when 2
+    - when 'ECS'
+      - case workshop.phase_short_name
+        - when 'Phase 2'
           %a{href:"https://docs.google.com/a/code.org/forms/d/12-HzFgPn6vP-D5P1XCkYxg_Dv3_DWM0B-xh1VDBuhNo/viewform"}
```

Comment:
That would be nice. Each URL is associated with a combination of program_type and phase, so it can't go directly in either of those existing hashes. It seems it would still have to be either be a nested case statement, or a nested hash, and I'm not sure we'd gain much in readability. Thoughts?